pom.xml

```diff
@@ -517,6 +517,12 @@
                 <artifactId>zstd-jni</artifactId>
                 <version>${zstd-jni.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.tukaani</groupId>
+                <artifactId>xz</artifactId>
+                <version>1.9</version>
+                <scope>compile</scope>
+            </dependency>
```

Comment:
Indicate the version number like `${xz.version}` ?

Reply:
ok, revised it